tests/unit/testplan/testing/multitest/entries/test_base.py

```diff
@@ -116,3 +117,22 @@ def test_summary():
 
     assert len(alpha_category_less_passing.entries) == summary.num_passing
     assert len(alpha_category_less_failing.entries) == summary.num_failing
+
+
+def test_TableLogRenderer():
+    dict_of_types = {
+        "bytes": "@βcdé".encode("UTF-8"),
+        "int": 24,
+        "string": "something",
+    }
+    list_of_lists = [[k, v] for k, v in dict_of_types.items()]
+    list_of_lists.insert(0, ["type", "value"])
```

Comment:
Good with me, I liked the addition more, it is more efficient (not here, in general).